docs/root/configuration/cluster_manager/cluster_hc.rst

```diff
@@ -14,52 +14,26 @@ Health checking
 TCP health checking
 -------------------
 
-.. attention::
-
-  This section is written for the v1 API but the concepts also apply to the v2 API. It will be
-  rewritten to target the v2 API in a future release.
-
-The type of matching performed is the following (this is the MongoDB health check request and
-response):
+The type of matching performed is the following:
 
 .. code-block:: json
 
   {
     "send": [
-      {"binary": "39000000"},
-      {"binary": "EEEEEEEE"},
-      {"binary": "00000000"},
-      {"binary": "d4070000"},
-      {"binary": "00000000"},
-      {"binary": "746573742e"},
-      {"binary": "24636d6400"},
-      {"binary": "00000000"},
-      {"binary": "FFFFFFFF"},
-      {"binary": "13000000"},
-      {"binary": "01"},
-      {"binary": "70696e6700"},
-      {"binary": "000000000000f03f"},
-      {"binary": "00"}
+      {"text": "39000000"},
+      {"text": "EEEEEEEE"},
      ],
      "receive": [
-      {"binary": "EEEEEEEE"},
-      {"binary": "01000000"},
-      {"binary": "00000000"},
-      {"binary": "0000000000000000"},
-      {"binary": "00000000"},
-      {"binary": "11000000"},
-      {"binary": "01"},
-      {"binary": "6f6b"},
-      {"binary": "00000000000000f03f"},
-      {"binary": "00"}
+      {"text": "EEEEEEEE"},
+      {"text": "01000000"},
```

Comment:
Why not switch to YAML?

Reply:
Partially because this is a config snippet, partially because I'm trying to do the minimal work to remove the deprecated docs